check_osp_main.py

```diff
@@ -99,7 +99,13 @@ def main():
     # set logger
     logger = get_logger(args.local)
     if float(args.warning) > float(args.critical):
-        print("Error: warning value can not be greater than critical value")
+        logger.error("Error: warning value can not be greater than critical value")
+        sys.exit(3)
+    elif (args.warning is None and args.critical is not None) or \
```

Comment:
Should've commented here as well, I meant both of the conditions with that previous statement. This can be removed now, since both will have default values.